R/assert-that.r

```diff
@@ -87,6 +87,8 @@ check_result <- function(x) {
     stop("assert_that: assertion must return a logical value", call. = FALSE)
   if (any(is.na(x)))
     stop("assert_that: missing values present in assertion", call. = FALSE)
+  if (length(x) == 0)
+    stop("assert_that: assertion has length 0", call. = FALSE)
   if (length(x) > 1) {
```

Comment:
How about just making this `length(x) != 1` and updating the message accordingly